### PR TITLE
chore: Janitorial work in foundation

### PR DESCRIPTION
**Pull request by fredrik-bakke**
This is a follow-up to #1085 with changes from #1032.

### CODE REVIEW

File: src/foundation/action-on-equivalences-functions.lagda.md [hunk 1]
```diff
@@ -24,17 +24,21 @@ open import foundation-core.identity-types
...
+  action-equiv-function f : X ≃ Y → f X ≃ f Y.
+```
 
 Alternatively, one can apply
 [transport along identifications](foundation-core.transport-along-identifications.md)
 to get
-[transport along equivalences](foundation.transport-along-equivalences.md), but
-luckily, these two notions coincide.
+[transport along equivalences](foundation.transport-along-equivalences.md), and
+by uniqueness these two constructions coincide.
```
Comment (VojtechStep) [inline src/foundation/action-on-equivalences-functions.lagda.md:41]: It would be nice to at least add a hint to where this uniqueness comes from

File: src/foundation/diagonals-of-maps.lagda.md [hunk 2]
```diff
@@ -34,56 +34,45 @@ open import foundation-core.truncation-levels
...
+    fiber (diagonal-map f) t → fiber (ap f) (pr2 (pr2 t))
+  fiber-ap-fiber-diagonal-map (z , refl) = (refl , refl)
+
+  fiber-diagonal-map-fiber-ap :
+    fiber (ap f) (pr2 (pr2 t)) → fiber (diagonal-map f) t
+  fiber-diagonal-map-fiber-ap (refl , refl) = (pr1 t , refl)
+
+  abstract
+    is-section-fiber-diagonal-map-fiber-ap :
+      fiber-ap-fiber-diagonal-map ∘ fiber-diagonal-map-fiber-ap ~ id
```
Comment (VojtechStep) [inline src/foundation/diagonals-of-maps.lagda.md:60]: Could you rephrase these with `is-section` and `is-retraction`?

File: src/foundation/action-on-equivalences-functions.lagda.md [hunk 3]
```diff
@@ -24,17 +24,21 @@ open import foundation-core.identity-types
...
 ## Idea
 
-Applying the
+Assume given a map between universes `f : 𝒰 → 𝒱`. Applying the
 [action on identifications](foundation.action-on-identifications-functions.md)
 to [identifications](foundation-core.identity-types.md) arising from the
-[univalence axiom](foundation.univalence.md) gives us the **action on
-equivalences**.
+[univalence axiom](foundation.univalence.md) gives us the {{#concept "action on
+equivalences" Agda=action-equiv-function}}
```
Comment (VojtechStep) [inline src/foundation/action-on-equivalences-functions.lagda.md:31]: I'm 99% sure that this `{{#concept}}` macro being split on multiple lines should have caused the CI to fail, but it didn't 🤔

**Review by VojtechStep — COMMENTED** [review]
LGTM 👌

**Comment by fredrik-bakke** [comment]
I've addressed your comments now @VojtechStep. Let me know if it is mergeable!

File: src/orthogonal-factorization-systems/lifting-structures-on-squares.lagda.md [hunk 4]
```diff
@@ -72,17 +72,21 @@ consists of a
...
 ```
 
 Therefore we see that a lifting square consists of a
 [morphism of arrows](foundation.morphisms-arrows.md) `α : hom-arrow f g` from
 `f` to `g`, a map `j : B → X`, and a
 [homotopy of morphisms of arrows](foundation.homotopies-morphisms-arrows.md)
-`pullback-hom f ~ α`.
+`pullback-hom f g j ~ α`.
+
+**Note.** In the literature, a lifting structure on a square is more commonly
```
Comment (fredrik-bakke) [inline src/orthogonal-factorization-systems/lifting-structures-on-squares.lagda.md:88]:
```suggestion
**Terminology.** In the literature, a lifting structure on a square is commonly
```

**Comment by VojtechStep** [comment]
It is very mergeable.
